os_migrate/plugins/module_utils/server.py

```diff
@@ -67,17 +67,25 @@ class Server(resource.Resource):
             'volume_type': None,
         },
         'floating_ip_mode': 'auto',
-        'port_creation_mode': 'nova'
+        'port_creation_mode': 'nova',
+        'boot_volume_id': None,
+        'data_copy': True,
+        "additional_volume_ids": None
     }
 
     @classmethod
     def from_sdk(cls, conn, sdk_resource):
         obj = super(Server, cls).from_sdk(conn, sdk_resource)
         params = obj.params()
         migration_params = obj.migration_params()
+        from_image = params.get('image_ref')
+
+        # booted from local boot disk, boot-disk not copied
+        if migration_params['data_copy'] is False and from_image is not None:
+            migration_params['boot_disk_copy'] = False
 
         # boot-from-volume servers must copy boot disk
-        if params.get('image_ref') is None:
+        if from_image is None:
```

Comment:
We probably don't need to change anything here, as in from_sdk migration params will always be defaulted, so 'data_copy' will never be False.